lib/src/gantt_default_day_header.dart

```diff
@@ -3,36 +3,49 @@ import 'package:flutter/material.dart';
 import 'week_day.dart';
 
 class GanttChartDefaultDayHeader extends StatelessWidget {
+  final Color? color;
+  final Color? backgroundColor;
```

Comment:
I think that these should be defined as functions that take `isHoliday` as parameter

Reply:
or not , since it's passed as a `bool` now, sorry

Reply:
> I think that these should be defined as functions that take `isHoliday` as parameter

Yeah, I moved the isHoliday logic to the consumer but it makes sense both ways to me...